routes/v1/user_inputs.routes.js

```diff
@@ -0,0 +1,14 @@
+var router = require('express').Router();
+const Injector = require('../../config/injector')
+const userInputsController = Injector.UserInputsController;
+
+router.get('/user_inputs',  userInputsController.listUserInputs);
+router.post('/user_inputs', userInputsController.createUserInput);
+router.get('/user_inputs/experiment/:experiment_id', userInputsController.listUserInputsByExperiment);
+router.get('/user_inputs/device/:device_id', userInputsController.listUserInputsByDevice);
+router.get('/user_inputs/:id', userInputsController.getOneUserInput);
+router.delete('/user_inputs/:id', userInputsController.deleteUserInput);
+router.post('/user_inputs/description', userInputsController.changeDecsription);
```

Comment:
same as in the controller: typo in description

Reply:
Seen, will correct everywhere